pyshelf/cloud/factory.py

```diff
@@ -13,10 +12,6 @@ def create_storage(self, bucket_name):
         # required and is not used because we want the ability to change
         # buckets when we want.
         bc = utils.get_bucket_config(self.config, bucket_name)
-        if bc is None:
```

Comment:
I put a note on the other pull request about this. Since bulk update drops any non-existant buckets, what would use this?